src/scripts/main.js

```diff
@@ -357,4 +357,24 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard tbody');
+
+people.forEach((person) => {
+  const age = person.died - person.born;
+  const century = Math.ceil(person.died / 100);
```

Comment:
The calculation for the century is correct, but ensure that this is the intended logic. It rounds up to the nearest century, which is typically correct for historical data.